ember-cli/2017-08/august-24.md

```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-08-24
+
+## Attendees
+
+
+## Agenda
```

Comment:
Changes to default addon blueprint:

* Move `./index.js` to `lib/index.js`
* Add [eslint-plugin-node](https://github.com/mysticatea/eslint-plugin-node) (and `extends: 'plugin:node/recommended'`)
* Add basic node-land QUnit test harness out of the box